### PR TITLE
Getting Started Instructions for certified partners

### PR DESCRIPTION
**Pull request by v-pravek**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@v-pravek__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-pravek). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: doc/get_started/windows-10-iot-core-Keith-Koep_myon-csharp.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
```
Comment (tameraw) [inline doc/get_started/windows-10-iot-core-Keith-Koep_myon-csharp.md]: What is that name "Keith" in the file?

**Comment by ashokkhurana** [comment]
Kieth and Koep is the company name and Myon is their device name.

**Comment by v-pravek** [comment]
Got the file name corrected.

**Pull request merged**